docs/sources/datasources/loki.md

```diff
@@ -54,92 +48,65 @@ You can use a debug section to see what your fields extract and how the URL is i
-
-The Logs Explorer (the `Log labels` button) next to the query field shows a list of labels of available log streams. An alternative way to write a query is to use the query field's autocomplete - you start by typing a left curly brace `{` and the autocomplete menu will suggest a list of labels. Press the `Enter` key to execute the query.
-
-Once the result is returned, the log panel shows a list of log rows and a bar chart where the x-axis shows the time and the y-axis shows the frequency/count.
-
-<div class="medium-6 columns">
-  <video width="800" height="500" controls>
-    <source src="/assets/videos/explore_loki.mp4" type="video/mp4">
-    Your browser does not support the video tag.
-  </video>
-</div>
-
-<br />
-
-### Log Stream Selector
-
-For the label part of the query expression, wrap it in curly braces `{}` and then use the key value syntax for selecting labels. Multiple label expressions are separated by a comma:
-
-`{app="mysql",name="mysql-backup"}`
+| Name               | Description                                                                                                                                                                                                 |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Query expression` | Loki query expression, check out the [LogQL documentation](https://grafana.com/docs/loki/latest/logql/).                                                                                                    |
+| `Query type`       | Choose the type of query you would like to run. An instant query queries against a single point in time. We are using "To" time from the time range. A range query queries over the selected range of time. |
+| `Line limit`       | Upper limit for number of log lines returned by query. The default is the Maximum lines limit set in Loki settings.                                                                                         |
+| `Legend`           | Available only in Dashboard. Controls the name of the time series, using name or pattern. For example `{{hostname}}` is replaced with the label value for the label `hostname`.                             |
 
-The following label matching operators are currently supported:
+### Log browser
 
-- `=` exactly equal.
-- `!=` not equal.
-- `=~` regex-match.
-- `!~` do not regex-match.
+// TODO David
 
-Examples:
+## Querying with Loki
 
-- `{name=~"mysql.+"}`
-- `{name!~"mysql.+"}`
+There are two types of LogQL queries:
 
-The [same rules that apply for Prometheus Label Selectors](https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors) apply for Loki Log Stream Selectors.
+- **Log queries** that return the contents of log lines.
```

Comment:
```suggestion
- **Log queries:** returns the contents of log lines.
```

Reply:
Log queries do not need to be bold.

Reply:
Removed bold. Plus this was changed to `Log queries -`